test/internet/test-dgram-connect.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const common = require('../common');
+const { addresses } = require('../common/internet');
+const assert = require('assert');
+const dgram = require('dgram');
+
+const PORT = 12345;
```

Comment:
Nit: This variable can (and probably should) be removed and `common.PORT` used instead.

Reply:
Fixed in https://github.com/nodejs/node/pull/27565/commits/5369cd0fad3bc0d0ab4f392193600ef821c8486d .
Thank you for the feedback.